index.html

```diff
@@ -0,0 +1,211 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/085384e239.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+</head>
+
+<body>
+    <!----------navbar setion------------------->
+    <nav class="nav-container">
+        <div class="sidebar-right" id="sidebarright">
+            <div class="cross"><a href=""><i class="fa-sharp fa-solid fa-circle-xmark"></i></a></div>
+            <div><a href="">Portfolio</a></div>
+            <div><a href="">About</a></div>
+            <div><a href="">Contact</a></div>
+
+        </div>
+        <h1 class="item-logo">Hyder</h1>
```

Comment:
- [ ] I think this is more like a logo, especially because this is not the title of our page. Please consider using an `a` for this logo. This makes you adopt HTML/CSS best practices, as well as use semantic tags.